src/app/components/drawer/Projects/SettingsComponent.js

```diff
@@ -0,0 +1,244 @@
+import { can } from '../../Can';
+import UserUtil from '../../user/UserUtil';
+import { withSetFlashMessage } from '../../FlashMessage';
+import styles from './Projects.module.css';
+
+const messages = defineMessages({
+  annotations: {
+    id: 'teamSettingsNavigation.annotations',
+    defaultMessage: 'Annotations',
+    description: 'Label for the Annotations settings navigation menu item',
+  },
+  columns: {
+    id: 'teamSettingsNavigation.columns',
+    defaultMessage: 'Columns',
+    description: 'Label for the Columns settings navigation menu item',
+  },
+  data: {
+    id: 'teamSettingsNavigation.data',
+    defaultMessage: 'Data',
+    description: 'Label for the Data settings navigation menu item',
+  },
+  integrations: {
+    id: 'teamSettingsNavigation.integrations',
+    defaultMessage: 'Integrations',
+    description: 'Label for the Integrations settings navigation menu item',
+  },
+  languages: {
+    id: 'teamSettingsNavigation.languages',
+    defaultMessage: 'Languages',
+    description: 'Label for the Languages settings navigation menu item',
+  },
+  members: {
+    id: 'teamSettingsNavigation.members',
+    defaultMessage: 'Members',
+    description: 'Label for the Members settings navigation menu item',
+  },
+  newsletter: {
+    id: 'teamSettingsNavigation.newsletter',
+    defaultMessage: 'Newsletter',
+    description: 'Label for the Newsletter settings navigation menu item',
+  },
+  reports: {
+    id: 'teamSettingsNavigation.reports',
+    defaultMessage: 'Reports',
+    description: 'Label for the Reports settings navigation menu item',
+  },
+  rules: {
+    id: 'teamSettingsNavigation.rules',
+    defaultMessage: 'Rules',
+    description: 'Label for the Rules settings navigation menu item',
+  },
+  settings: {
+    id: 'teamSettingsNavigation.settings',
+    defaultMessage: 'Settings',
+    description: 'Header for the settings navigation menu items',
+  },
+  similarity: {
+    id: 'teamSettingsNavigation.similarity',
+    defaultMessage: 'Similarity',
+    description: 'Label for the Similarity settings navigation menu item',
+  },
+  statuses: {
+    id: 'teamSettingsNavigation.statuses',
+    defaultMessage: 'Statuses',
+    description: 'Label for the Statuses settings navigation menu item',
+  },
+  tags: {
+    id: 'teamSettingsNavigation.tags',
+    defaultMessage: 'Tags',
+    description: 'Label for the Tags settings navigation menu item',
+  },
+  tipline: {
+    id: 'teamSettingsNavigation.tipline',
+    defaultMessage: 'Tipline',
+    description: 'Label for the Tipline settings navigation menu item',
+  },
+  workspace: {
+    id: 'teamSettingsNavigation.workspace',
+    defaultMessage: 'Workspace',
+    description: 'Label for the Workspace settings navigation menu item',
+  },
+});
+
+const SettingsComponent = ({
+  team,
+  params,
+  intl,
+}) => {
+  const { tab } = params;
+  const userRole = UserUtil.myRole(window.Check.store.getState().app.context.currentUser, team.slug);
```

Comment:
I got this from https://github.com/meedan/check-web/blob/da765780169f0bc1e4357f84a0c2bcf3e98f9345/src/app/components/Sandbox.js#L335 as this seemed the way to get this information in a function component but I am unsure how to fix the unit test in the PR